os/net/lwip/test/unit/mbox/mbox_issue.h

```diff
@@ -0,0 +1,57 @@
+/****************************************************************************
+ *
+ * Copyright 2021 Samsung Electronics All Rights Reserved.
```

Comment:
Please change the year to 2022.
Happy New Year :)

Reply:
Thanks, Happy New Year :)